src/MainWindow.py

```diff
@@ -312,11 +312,11 @@ def DefineResourcesMenu(self) -> None:
 
     def DefineTestingMenu(self) -> None:
         self.Button_InputField = self.ButtonTemplate(
-            "User Input", self.Button_InputField_func)
+            translations["test_user_input"], self.Button_InputField_func)
         self.PopupBox_gui = self.ButtonTemplate(
-            "Popup Box", self.PopupBox_test_func)
+            translations["test_popup_box"], self.PopupBox_test_func)
         self.Button_PrintToConsole = self.ButtonTemplate(
-            "Print to Console", self.Button_PrintToConsole_func)
+            translations["test_print_to_console"], self.Button_PrintToConsole_func)
```

Comment:
for the "test buttons" i'd like if they stay english only so we don't need to add new values in the translation files every time we want to quickly debug something